modules/cli/pkg/launcher/remoteLauncher.go

```diff
@@ -85,14 +85,12 @@ func (launcher *RemoteLauncher) SubmitTestRun(
 	testRunRequest := galasaapi.NewTestRunRequest()
 	testRunRequest.SetClassNames(classNames)
 	testRunRequest.SetRequestorType(requestType)
-	testRunRequest.SetRequestor(requestor)
+	testRunRequest.SetRequestor(systemUser)
```

Comment:
If the API server uses the requestor in the JWT, do we still need to set the requestor in the request payload?

Reply:
Fixed in 1d1e00f96513ca8cdd74b42e40eae0751e30f40b